doc/GETTING_STARTED.md

```diff
@@ -0,0 +1,428 @@
+
+#### Peer Routing
+
+Peer Routing offers a way to find other peers in the network by issuing queries using a Peer Routing algorithm, through their peer ids.
+
+Looking at the [available peer routing](./CONFIGURATION.md#peer-routing) protocols, we might choose `libp2p-kad-dht` or `libp2p-delegated-peer-routing`. If your runtime is a browser or a low power device, you might prefer to use `libp2p-delegated-peer-routing` as you are able to delegate other peer to issue the queries for you. However, you need to know a peer able to do that. Otherwise, you should use `libp2p-kad-dht`.
+
+In this context, we will install the dht. You should revisit the [API](./API.md) document, more specifically [API#peerRouting.findPeer](./API.md#peerRoutingfindPeer).
+
+```sh
+npm install libp2p-kad-dht
+```
+
+`js-libp2p` configuration is not well defined yet regarding peer routing. Therefore, the `dht` should be plugged through a `modules.dht` property, while the `libp2p-delegated-peer-routing` should be plugged into `modules.peerRouting` array.
+
+```js
+const Libp2p = require('libp2p')
+const TCP = require('libp2p-tcp')
+const MPLEX = require('libp2p-mplex')
+const SECIO = require('libp2p-secio')
+const DHT = require('libp2p-kad-dht')
+
+const node = await Libp2p.create({
+  modules: {
+    transport: [TCP],
+    streamMuxer: [MPLEX],
+    connEncryption: [SECIO],
+    dht: DHT
+  },
+  config: {
+    dht: { // The DHT options (and defaults) can be found in its documentation
+      enabled: true,
+      randomWalk: {
+        enabled: false, // dht discovery (enabled by default)
+        interval: 300e3,
+        timeout: 10e3
+      }
+    }
+  }
+})
+
+// start libp2p
+await node.start()
+
+// Find peer
+const peerInfo = await libp2p.peerRouting.findPeer(peerId)
+
+// ...
+```
+
+You can find out more about peer routing on the example provided on [./CONFIGURATION.md#setup-with-content-and-peer-routing](./CONFIGURATION.md#setup-with-content-and-peer-routing).
+
+<details><summary>Read More</summary>
+If you want to know more about libp2p peer routing, you should read the following content:
+
+- https://docs.libp2p.io/concepts/peer-routing
+</details>
+
+#### Content Routing
+
+Content routing provides a way to find where content lives in the network. It works in two steps: 1) Peers provide (announce) to the network that they are holders of specific content and 2) Peers issue queries to find where that content lives. A Content Routing mechanism could be as complex as a DHT or as simple as a registry somewhere in the network.
+
+Looking at the [available content routing](./CONFIGURATION.md#content-routing) protocols, we might choose `libp2p-kad-dht` or `libp2p-delegated-content-routing`. If of your runtime is a browser or a low power device, you might prefer to use `libp2p-delegated-peer-routing` as you are able to delegate other peer to issue the queries for you. However, you need to know a peer able to do that. Otherwise, you should use `libp2p-kad-dht`.
```

Comment:
```suggestion
Looking at the [available content routing](./CONFIGURATION.md#content-routing) modules, we might choose `libp2p-kad-dht` or `libp2p-delegated-content-routing`. If your runtime is a browser or a low power device, you might prefer to use `libp2p-delegated-content-routing` as you are able to delegate to another peer to issue the queries for you. However, you need to know a peer able to do that. Otherwise, you should use `libp2p-kad-dht`.
```